build.gradle

```diff
@@ -37,9 +34,17 @@ apply from: "$rootDir/gradle/idea.gradle"
 apply from: "$rootDir/gradle/tests.gradle"
 
 // The task for running example application from the command line.
-//#docregion run-task
 task sayHello(type: JavaExec) {
     main = 'io.spine.helloworld.Example'
     classpath = sourceSets.main.runtimeClasspath
 }
-//#enddocregion run-task
+
+// The task for composite builds to build this example.
+//
+// This build task is not a part of the example. It is used by the composite build of
+// the `spine.io` project which includes this project for embedding the source code of
+// this example into the site.
+//
+task buildAll(type: GradleBuild) {
```

Comment:
Why do we use Proto-style comments in build scripts?

Gradle scripts are basically Groovy scripts. And Groovy from my perspective inherits from Java in terms of the code style and Javadocs.

Reply:
The build tasks are not methods. Groovy sits on JVM, but Gradle is a way too far from Java as a programming paradigm. I don't want to have a mix with comment styles here and comments that come to the _real_ parts of the example.

Reply:
After further thought, I think I'll change it to be Javadoc-like. It would make it stand apart.